### ISSUE TITLE
Loconotion timing out due to weird "notion-unknown-block" div

### ISSUE THREAD

**Issue by kmlbgn** [leoncvlt/loconotion#129]
There seems to be a bug in notion that spawns a persistent "notion-unknown-block" div which stalls the notion_page_loaded() call.
I am unable to recreate this bug at will, and the only way to delete this div is by selecting a few lines around it and ctrl+x. 
Deleting doesn't work and is a signature of this bug.

![Capture d’écran 2023-01-14 004526](https://user-images.githubusercontent.com/59659182/212375355-2c7da46d-9e2c-41f5-b2f1-6f96658a614e.png)

**Comment by druidvav** [comment]
got exactly same problem

**Comment by sueszli** [comment]
Yes - there seem to be infinite loading spinners in Notion now that don't disappear no matter how long you wait.

I also was confronted with the same issue and friends and I resolved it in the refactored version / fork of loconotion which can be found here: https://github.com/sueszli/notionSnapshot/
